### ISSUE TITLE
Only in Idea plugin: LeftCurly "nl" doesn't work on METHOD_DEF

### ISSUE THREAD

**Issue by nator** [jshiell/checkstyle-idea#96]
Chased this on the main repo for a while, looks like it's Idea-specific
https://github.com/checkstyle/checkstyle/issues/112

This rule seems to work for all other tokens (if, while, class definition) but doesn't catch method definitions with a curly brace on the same line.

```
<module name="LeftCurly">
    <property name="option" value="nl"/>
</module>
```

Simplest class to repro:

```
public class TestClass
{
    public int testMethod() {
        return 0;
    }
}
```
